src/items.py

```diff
@@ -1,14 +1,20 @@
 import logging
 
-from typing import List
-from collections import namedtuple
+from typing import Set
+from collections import namedtuple, defaultdict
 
 log = logging.getLogger()
 
 
-# Structure for item: Each item has a name and id
-Item = namedtuple('Item', ['name', 'id'])
-# Item requirement - Item, Quantity
+'''
+    Item represents a single indivisible production unit.
+    type: there can be many units of the same type. e.g. blue-door
+    id: unique to the specific instance of the item. e.g. YHG87
+        there can only be one item per id and given type
+'''
+Item = namedtuple('Item', ['type', 'id'])
```

Comment:
Can we not rename basic structures if it's not critical? This could affect hanging diffs plus silently break untested code.

Reply:
I felt the same way. But, name did not make sense and I made replacements anywhere that was necessary. Will avoid future changes of this kind.